detect_secrets/plugins/github_enterprise.py

```diff
@@ -8,9 +8,9 @@
 
 
 class GheDetector(RegexBasedDetector):
-    """ Scans for GitHub credentials """
+    """ Scans for GitHub Enterprise credentials """
 
-    secret_type = 'GitHub Credentials'
+    secret_type = 'GitHub Enterprise Credentials'
```

Comment:
I looked into seeing if existing baselines using old secret type would get break or auto upgrade. I have found that it will auto update the secret type from `GitHub Credentials` to `GitHub Enterprise Credentials` automatically. 

Do we need to wait to make this change at the same time as detect-secrets-stream? Or can this pr be merged and then dss can be updated at a later time?

Reply:
> Do we need to wait to make this change at the same time as detect-secrets-stream? 

No, it does not need to happen at the same time.

> Or can this pr be merged and then dss can be updated at a later time?

Yes, this can merge and DSS can pick it up later.

Reply:
> I have found that it will auto update the secret type from GitHub Credentials to GitHub Enterprise Credentials automatically.

Does it also migrate the audit status? or just drop all the old ones and detect the same ones as new leaks under new type?

Reply:
Yes, it keeps the audit status! The only thing that gets dropped/updated is the token type, everything else stays the same

Reply:
That's cool. Given that I'm good with merging.